data/json/items/comestibles/med.json

```diff
@@ -1210,7 +1210,6 @@
         "id" : "gummy_vitamins",
         "copy-from": "vitamins",
         "type" : "COMESTIBLE",
-        "comestible_type" : "MED",
```

Comment:
I have some doubts about purpose of ```"comestible_type" : "MED",``` but I wasn't able to find any difference with and without this line so I'm going to merge this PR soon.